src/main/java/at/ac/tuwien/kr/alpha/common/predicates/Evaluable.java

```diff
@@ -0,0 +1,16 @@
+package at.ac.tuwien.kr.alpha.common.predicates;
+
+import at.ac.tuwien.kr.alpha.common.terms.ConstantTerm;
+import at.ac.tuwien.kr.alpha.common.terms.Term;
+
+import java.util.List;
+import java.util.Set;
+
+import static java.util.Collections.*;
+
+public interface Evaluable extends Predicate {
```

Comment:
How about renaming `Evaluable` to `InterpretationAgnosticPredicate` (or `FixedValuePredicate`)?

Reply:
Renamed it to `FixedInterpretationPredicate`, hope that is fine.